src/components/previewCard/imageHelper.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from "react"
+
+import styles from "./index.module.css"
+
+interface Props {
+  imageSrc: string
+  pictureAlt: string
+  isLogo: boolean
+  handleOnErrorImage?: (e) => void
+}
+
+const Image: FC<Props> = ({
+  imageSrc,
+  pictureAlt,
```

Comment:
```suggestion
  imageAlt,
```

otherwise I'm thinking that a `picture` is something different from an `image` :)